src/resources/Limits/Limits.ts

```diff
@@ -1,10 +1,11 @@
 import API from '../../APICore';
 import {LicenseSection} from '../Enums';
 import Resource from '../Resource';
-import {AllLimitsModel, LimitHistoryDataPointModel, LimitsModel} from './LimitsInterfaces';
+import {AllLimitsModel, LimitHistoryDataPointModel, LimitsModel, LimitType} from './LimitsInterfaces';
 
 export default class Limits extends Resource {
     static getBaseUrl = () => `/rest/organizations/${API.orgPlaceholder}/limits`;
+    static getWithLimitType = (limitType: LimitType) => `${Limits.getBaseUrl()}?limitType=${limitType}`;
```

Comment:
You should use https://github.com/coveo/platform-client/blob/6b8ce4dde75754cd127d1239431edcd1387cb58a/src/resources/Resource.ts#L10 for the query params stuff.

Reply:
Done, thx for the cue, I figured encoding didn't matter since the enum is url safe but that's definitely better.